_config.yml

```diff
@@ -1,1827 +1,1827 @@
-      is proudly maintained by members of the Ruby community.
-    es: |
-      Este sitio web está desarrollado con Ruby y <a href="http://www.jekyllrb.com/">Jekyll</a>
-      y es orgullosamente mantenido por miembros de la comunidad.
-      Si quieres contribuir, por favor hazlo en <a href="https://github.com/ruby/www.ruby-lang.org/">GitHub</a>
-      o contacta a nuestro <a href="mailto:webmaster@ruby-lang.org">webmaster</a>
-      si tienes preguntas o comentarios.
-    fr: |
-      Ce site est propulsé par Ruby et <a href="http://www.jekyllrb.com/">Jekyll</a>.
-      Il est fièrement maintenu par des membres de la communauté Ruby.
-      Contribuez en vous rendant sur <a href="https://github.com/ruby/www.ruby-lang.org/">GitHub</a>
-      ou contactez le <a href="mailto:webmaster@ruby-lang.org">webmaster</a>
-      pour toute question ou tout commentaire.
-    id: |
-      <a href="/id/about/website/">Situs ini</a>
-      dengan bangga dirawat oleh anggota komunitas Ruby.
-    it: |
-      <a href="/it/about/website/">Questo sito web</a>
-      è mantenuto con orgoglio dai membri della comunità Ruby.
-    ja: |
-      <a href="/ja/about/website/">このウェブサイト</a>は Ruby コミュニティのメンバーによって運営されています。
-    ko: |
-      <a href="/ko/about/website/">이 사이트</a>는 루비 커뮤니티의
-      구성원들이 직접 관리합니다.
-    pl: |
-      <a href="/pl/about/logo/">Logo Rubiego</a> jest Zastrzeżone &copy; 2006, Yukihiro Matsumoto; licencjonowane na warunkach <a href="http://creativecommons.org/licenses/by-sa/2.5/">CC BY-SA 2.5</a>.<br><br>
-      Serwis powstał w języku Ruby przy użyciu <a href="http://www.jekyllrb.com/">Jekyll</a>.
-      Utrzymaniem serwisu zajmuje się społeczność Rubiego.
-      Projekt strony <a href="https://twitter.com/jasonzimdars">Jason Zimdars</a>.
-      Proszę udziel nam wsparcia na <a href="https://github.com/ruby/www.ruby-lang.org/">GitHub</a>
-      lub skontaktuj się z naszym <a href="mailto:webmaster@ruby-lang.org">webmasterem</a>
-      jeśli masz jakieś pytania lub komentarze dotyczące tego serwisu.
-    pt: |
-      <a href="/pt/about/website/">Este website</a>
-      é mantido com orgulho por membros da comunidade Ruby.
-    ru: |
-      <a href="/ru/about/website/">Этот сайт</a> с гордостью поддерживается членами сообщества Ruby.
-    tr: |
-      <a href="/en/about/website/">Bu web sitesi</a>
-      Ruby topluluğunun üyeleri tarafından gururla sürdürülmektedir.
-    vi: |
-      Website này được tạo ra với Ruby sử dụng <a href="http://www.jekyllrb.com/">Jekyll</a>.
-      Trang được bảo trì bởi các thành viên của cộng đồng Ruby.
-      Xin đóng góp trên <a href="https://github.com/ruby/www.ruby-lang.org/">GitHub</a>
-      hay liên lạc với <a href="mailto:webmaster@ruby-lang.org">webmaster</a> (bằng tiếng Anh)
-      nếu có câu hỏi hoặc ý kiến phản hồi về trang này.
-    zh_cn: |
-      <a href="/zh_cn/about/website/">本站</a>由 Ruby 社区的成员共同维护。
-    zh_tw: |
-      <a href="/zh_tw/about/website/">本站</a>由 Ruby 社群成員共同維護。
```

Comment:
It seems like the incoming changes on this file are exactly similar to the ones already exist, what I would do it to get this change out of the way and try to merge again.
cc @Goudarz

Reply:
BTW this can be simply because of line endings difference between the file on the source and the one on the destination, I've noticed this kind of issue when using some special auto manipulation tools provided by editors(IDEs/Text editor, etc)